### ISSUE TITLE
Need to find a solution on how to name paired fastq files

### ISSUE THREAD

**Issue by cokelaer** [sequana/sequana#526]
currently, we assume that users have the tag R1 / R2 
In sequana pipelines, we allows other pattern such as _1, _2 but this lead to issues in some cases because we search for _R1 / _R2 and count them. The reason is that we cannot know before hand whether the _R1 is at the end or not. So, a file named tata_R1_titi_R1 shows the tag R1 twice. A better example is when the _R is bot present. Consider:

    test_1_1
    test_1_2

Here, we have a paired files indicated by the suffixes _1 and _2; the sample is called 1 so if we search for _1, we will find it threed times and _2 only once and the software in snaketools is currently saying that this is unpaired. Once solution is to not allow people to analyse files that do not have the R1/R2 tag and this will simplify our life .

**Comment by ddesvillechabrol** [comment]
The simplest way to bind R1 and R2 file is to use a sample_plan as CSV file.
We can may create a widget to generate this CSV file or use one already done.
Like:
```
sample_name,R1,R2
toto,toto.1.fq,toto.2.fq
tutu,tutu.R1.fastq.gz,tutu.R2.fastq.gz
```

**Comment by cokelaer** [comment]
no need for now. In all pipelines, we now use the read tag. Most sequencers produces _R1_ _R2_ tag. Some compagnies have the tag at the end e.g. tata_1.fastq ; in that was we can use the "_[12]." pattern with the dot. Others who will use tata_1_1_L001 for example will not be handle for now
